test/bugs-reported/bug-descriptions

```diff
@@ -0,0 +1,143 @@
+*****************************************
```

Comment:
I don't think this file is necessary given the contents of the .future files.

Also, I think we would prefer a different directory for these bugs. While you might be able to find an appropriate existing directory, it might be easiest to create test/users/rohanbadlani/bugs/ instead.